bees/sunbee/sunbeefactory.go

```diff
@@ -63,11 +63,18 @@ func (factory *SunBeeFactory) Image() string {
 func (factory *SunBeeFactory) Options() []bees.BeeOptionDescriptor {
 	opts := []bees.BeeOptionDescriptor{
 		{
-			Name:        "city",
+			Name:        "query",
 			Description: "The name of the city where the sunrise/sunset will happen",
 			Type:        "string",
 			Mandatory:   true,
 		},
+		{
+			Name:        "offset",
+			Description: "Fire the event this number of seconds before sunset/sunrise (default: 2 min)",
+			Type:        "int",
+			Default:     120,
```

Comment:
Maybe we should keep the default offset at 0 and make this non-mandatory? I don't think it needs to be enforced.

Reply:
Done. Bear in mind that the check interval is 60 secs so we'll always have a +/- 60 secs offset/error.